demo/src/main/java/com/example/demo/DemoApplicationForSQ.java

```diff
@@ -5,4 +5,11 @@
 	public void doSomething() {
 	}
 
+	public void doSomething2() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)